roles/StackStorm.mongodb/tasks/mongodb_auth.yml

```diff
@@ -1,6 +1,14 @@
 ---
-- name: Ensure pymongo is installed for the mongodb_user ansible module
+- name: Install pip (for the installation of pymongo)
   become: yes
   package:
-    name: python-pymongo
+    name: python-pip
     state: present
+  tags: [databases, mongodb]
+
+- name: Install pymongo (for the mongodb_user module)
+  # pymongo 3.2+ is required to work with mongo 3.2+. system packages are too old.
+  become: yes
+  pip:
+    name: "pymongo=={{ mongodb_major_minor_version }}"
```

Comment:
Latest `pymongo` is `3.10.1` as for  `Jan 8, 2020 ` https://pypi.org/project/pymongo/

What will happen if `mongodb_major_minor_version` would be `4.0` if needed for Bionic #253 and EL8?
 
I don't think pymongo pip version correlates with `mongodb_major_minor_version`.

Reply:
Ah. I guess my sample size was too small when I looked that up. For versions 3.2-3.6, that assumption held.
Here's the chart showing supported versions: https://docs.mongodb.com/ecosystem/drivers/driver-compatibility-reference/#python-driver-compatibility
Now to find an elegant way to record those requirements.

Reply:
I think it's possible to use one of the latest versions pinned with safe interval.
My understanding that this should work for all older MongoDB versions.

Reply:
`pymongo>=3.10.1,<4.0.0`

Reply:
K. the latest commit pins the version as suggested.